.github/workflows/main.yml

```diff
@@ -92,20 +92,14 @@ jobs:
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: ${{ matrix.dotnet-version }}
-      - name: Run the app
+      - name: Run and Test the app
         run: |
           dotnet run --project DOTNET-example/Web-App &
-      - name: Sleep for 30 seconds
-        uses: jakejarvis/wait-action@master
-        with:
-          time: '30s'
```

Comment:
No need for a sleep with the tasks combined into a single run step? Is there a flaky race condition if the `dotnet run` command doesn't get the app up by the time we run `curl`?

Reply:
We do not need to sleep. This file is outdated.